security/cmd/node_agent_k8s/README.md

```diff
@@ -22,7 +22,7 @@ nodeagent-qt6z6                   1/1       Running       0          3m
 
 - Deploy the workload pod.
 ```shell
-kubectl apply -f workload/workload.yaml
```

Comment:
Since you have already change the code, maybe build a new version of docker for nodeagent, and use it in https://github.com/istio/istio/blob/master/security/cmd/node_agent_k8s/nodeagent.yaml#L22

Can be done in a follow-up PR

Reply:
Sure, I add a note in the issue description, https://github.com/istio/istio/issues/4190.